src/unitTest/PageParserTest.php

```diff
@@ -0,0 +1,40 @@
+<?php
+
+include_once '../parserPages/PageParser.php';
+include_once '../dataBase/XmlFeedModel.php';
+
+use PHPUnit\Framework\TestCase;
+
+class PageParserTest extends TestCase
+{
+
+    protected $objectPage;
+
+    public function setUp()
+    {
+        parent::setUp();
+        $this->objectPage = new PageParser();
+    }
+
+    public function testPageParser(): void
+    {
+        $sourceLink = 'https://usn.ubuntu.com/usn/rss.xml';
+        $haystack = $this->objectPage->getValuesInItem($sourceLink);
+        $this->assertIsArray($haystack);
+        $actual = $haystack[0];
+        $this->assertInstanceOf('XmlFeedModel', $actual, 'Failed asserting is not an instance of class');
+    }
+
+    public function testXmlFile(): void
```

Comment:
The name of the test should tell what is being tested. In your case its something like "testIncorrectXmlSource"